### PR TITLE
Added some more information about StackPanels

### PR DESCRIPTION
**Pull request by petterroea**

### CODE REVIEW

File: Fuse/04 - Layout.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Note that the element order in a `Panel` is the same as the layer order in popul
 
 ## $(StackPanel)
 
-The StackPanel places its children in a stack. The default layout is a vertical stack, but one can use the $(Orientation) property to specify that the stack should be layed out horizontally.
+The StackPanel places its children in a stack. The default layout is a vertical stack, but one can use the `Orientation` property to specify that the stack should be layed out horizontally.
```
Comment (yupferris) [inline Fuse/04 - Layout.md]: Why remove this link? Does it not work currently?

File: Fuse/04 - Layout.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ The StackPanel places its children in a stack. The default layout is a vertical
 </StackPanel>
 ```
 
+Additionally, one can use the `ItemSpacing` property to make some space between elements, as an alternative to setting a margin on the elements.
```
Comment (yupferris) [inline Fuse/04 - Layout.md]: Good call, mind adding an example as well?

File: Fuse/04 - Layout.md (hunk 1 above)
Comment (petterroea) [inline Fuse/04 - Layout.md]: The link didn't work, so i just replaced it with code formatting

Comment (sebbert) [inline Fuse/04 - Layout.md]: It looks like it wasn't supposed to be a link, but rather a marker for linking to Orientation from elsewhere. I'd suggest going for \`$(Orientation)\` instead, as removing the $() may break some links.

**Pull request merged**